polyfill/platform/ARKitWrapper.js

```diff
@@ -199,20 +222,101 @@ export default class ARKitWrapper extends EventHandlerBase {
 	Sends an addAnchor message to ARKit
 	Returns a promise that returns:
 	{
-		uuid - the anchor's uuid,
-		transform - anchor transformation matrix
+		uuid: DOMString,
+		worldTransform - anchor transformation matrix
 	}
 	*/
-	addAnchor(uid, transform){
+	addAnchor(name, transform){
```

Comment:
Why is this 'name' and removeAnchor uses 'uid'? Do they represent the same thing?

Reply:
Name is an optional and not unique inside IOS app. Uuids are internal and can't be changed. They are like automatic uuids in three.js. So uid and name represent different things.

To track updates for arbitrary anchors we can use uuid that we get when addAnchor promise resolves. And js names if they're specified in addAnchor are also attached to anchors.

Reply:
Perhaps we should change the WebXR APIs so that there is no name provided, the system (either the WebXR polyfill or the iOS layer) always provides the UID. Perhaps there no reason for the JS app to provide a name if it always has to use the UID to get the current info.

What do you think?

Reply:
I agree with you about removing name. In this case we need to do more changes in WebXR part that uses ARKitWrapper

Reply:
Yes, go ahead and make changes to WebXR.

Reply:
agreed, we should get the changes done in both the -polyfill and -ios repo's at the same time, into the dev branch.  When they work, we can move them into the master and ship an update to the app (assuming apple lets us publish)